ddpui/api/charts_api.py

```
@@ -1033,32 +1005,23 @@ def download_chart_data_csv(request, payload: ChartDataPayload):
 def get_chart(request, chart_id: int):
     """Get a specific chart"""
     orguser: OrgUser = request.orguser
+
     try:
-        chart = Chart.objects.get(id=chart_id, org=orguser.org)
-    except Chart.DoesNotExist:
+        chart = ChartService.get_chart(chart_id, orguser.org)
+    except ChartNotFoundError:
         raise HttpError(404, "Chart not found")
```

Ruff (0.14.8)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)